src/components/side-panel/layer-panel/layer-config-group.js

```diff
@@ -24,23 +24,35 @@ import styled from 'styled-components';
 import classnames from 'classnames';
 import {FormattedMessage} from 'react-intl';
 import Switch from 'components/common/switch';
-import InfoHelper from 'components/common/info-helper';
+import InfoHelperFactory from 'components/common/info-helper';
 import {VertThreeDots} from 'components/common/icons';
 
+const dashedPrependCss = `
```

Comment:
why do you need to add dash here? remove this

Reply:
removed